src/xcode/ENA/ENA/Source/Views/GradientView.swift

```diff
@@ -34,16 +34,56 @@ class GradientView: UIView {
 		case solidGrey
 		case lightBlue
 		case green
+		case lightBlueWithStars
+
+		var starImage: UIImage? {
+			switch self {
+			case .lightBlueWithStars:
+				return UIImage(imageLiteralResourceName: "stars")
+			default:
+				return nil
+			}
+		}
+
 	}
 
-	var type: GradientType = .blueRedTilted { didSet { setupLayer() } }
+	var type: GradientType = .blueRedTilted {
+		didSet {
+			setupLayer()
+			setupStars()
+		}
+	}
 
 	// MARK: - Private
 
+	private var withStars: Bool = false
+	private var starImageView: UIImageView = UIImageView()
+
+	private func setupStars() {
+		guard let image = type.starImage else {
+			starImageView.removeFromSuperview()
+			return
+		}
+
+		starImageView = UIImageView(image: image)
```

Comment:
we need to make sure UIImageView is nil before set an add it. If it's not nil there might be an old UIImageView, better remove that one from superview before assigning a new one.